canonical-web-and-design/vanilla-framework
# Add psuedo element on tooltip hover to allow interaction with tooltip message

#3269  Done

Added a small pseudo element to tooltip elements on hover, so that it's possible for a user to mouseover the tooltip message.

Fixes #2325 

## QA

- Pull code
- Run `./run`
- Open http://0.0.0.0:8101/docs/examples/patterns/tooltips or [demo]
- Hover over a tooltip button, see that you are able to move your cursor over the tooltip message without it disappearing.

## Screenshots

Before:
![tooltip1](https://user-images.githubusercontent.com/2376968/98561561-f37b7c00-22a0-11eb-8624-e99f4670d969.gif)

After:
![tooltip2](https://user-images.githubusercontent.com/2376968/98561577-f8403000-22a0-11eb-886c-fc173848924b.gif)
